### DISCUSSION TITLE
Switcher browser tabs

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16642] [General]
As a user I want to have my browser tabs included in the switcher so I can quickly switch between chatting with someone and visiting a site

### Designs:
https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?node-id=0%3A1

**Comment by du82** [comment]
@Parveshdhull Weren't you working on the switcher? This feature is still planned, right?

**Comment by Parveshdhull** [comment]
hi @du64, yes its still planned.
